### ISSUE TITLE
`bridgetown --help` raises an error outside of a Bridgetown project dir, if there's a `Gemfile`

### ISSUE THREAD

**Issue by davidstosik** [bridgetownrb/bridgetown#653]
I first commented on issue #608, as it felt the `cannot load such file -- rake` error was related enough, but then I realized it might be different enough, and the currently suggested fix did not change the outcome for me, so I decided to create an issue instead.

**Bridgetown Version**: 1.1.0

**To Reproduce**
```
mkdir /tmp/reproduce
cd /tmp/reproduce
rbenv local 3.1.2
gem install bridgetown
touch Gemfile
bridgetown --help
```

**Current behavior**
Bridgetown starts displaying the help text correctly, but when it comes to displaying the "Available Rake Tasks" sections, it craps out with an error:

```
bridgetown --help
Bridgetown v1.1.0 "Belmont" is a Webpack-aware, Ruby-powered static site generator for the modern Jamstack era

Usage:
  bridgetown <command> [options]

Commands:
  bridgetown apply                    # Applies an automation to the current site
  bridgetown build                    # Build your site and save to destination folder
  bridgetown clean                    # Clean the site (removes site output and metada...
  bridgetown configure CONFIGURATION  # Set up bundled Bridgetown configurations
  bridgetown console                  # Invoke an IRB console with the site loaded
  bridgetown dev                      # Alias of start
  bridgetown doctor                   # Search site and print specific deprecation war...
  bridgetown dream                    # There's a place where that idea still exists a...
  bridgetown esbuild ACTION           # Perform actions on the Bridgetown esbuild conf...
  bridgetown help <command>           # Show detailed command usage information and exit
  bridgetown new PATH                 # Creates a new Bridgetown site scaffold in PATH
  bridgetown plugins <command>        # List installed plugins or access plugin content
  bridgetown serve                    # DEPRECATED (Serve your site locally using WEBr...
  bridgetown start                    # Start the Puma server, frontend bundler, and B...
  bridgetown webpack ACTION           # Perform actions on the Bridgetown Webpack conf...

/Users/sto/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bridgetown-core-1.1.0/lib/bridgetown-core/commands/base.rb:97:in `require': cannot load such file -- rake (LoadError)
        from /Users/sto/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bridgetown-core-1.1.0/lib/bridgetown-core/commands/base.rb:97:in `help'
        from /Users/sto/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
        from /Users/sto/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
        from /Users/sto/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
        from /Users/sto/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor/base.rb:485:in `start'
        from /Users/sto/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bridgetown-core-1.1.0/bin/bridgetown:38:in `<top (required)>'
        from /Users/sto/.rbenv/versions/3.1.2/bin/bridgetown:25:in `load'
        from /Users/sto/.rbenv/versions/3.1.2/bin/bridgetown:25:in `<main>'
```

**Expected behavior**

When I'm getting started with Bridgetown (or say I forgot the name of the Bridgetown command to start a new site), it shouldn't matter where I'm situated, `bridgetown --help` should work without displaying an error. (Maybe skip displaying rake tasks? Maybe detect it's not running in a Bridgetown project directory, and just display help for `bridgetown new`, like Rails does?)

**Screenshots**
Not applicable.

**Computing environment (please complete the following information):**
 - OS: macOS
 - Browser: N/A
 - Browser Version: N/A
 - Ruby Version: 3.1.2 through rbenv

**Additional context**

I guess it doesn't matter much, I can get started and things should work once I `cd` into the site directory, but I thought I'd mention it. It doesn't feel great to get an error message when you're getting started. 😬

**Comment by davidstosik** [comment]
> If you open `irb` in that directory, what happens if you run `require 'rake'` in that `irb` console?

I'd tried that before posting, and it worked. I also confirmed with `gem list` and `ls` that the `rake` gem was available.
My guess is that maybe, similarly to #608, Bridgetown uses `bundle exec` somewhere, and because `rake` is not in my `Gemfile`, it is not made available to the command.

**Comment by davidstosik** [comment]
> Rake is bundled with ruby, see https://stdgems.org/3.1.0/

Sure, but how do you explain this?

```
mkdir /tmp/test
cd /tmp/test
rbenv local 3.1.2
bundle init
bundle install
bundle exec ruby -e "require 'rake'"
-e:1:in `require': cannot load such file -- rake (LoadError)
	from -e:1:in `<main>'
```

**Comment by davidstosik** [comment]
> Would you like to put together a PR for this issue?

I would have already, if I understood exactly what's going on.
I could test whether rake is present right before requiring it, I could rescue the error and bail early, but that sounds like a workaround at best.

The real question is: **why would that `require "rake"` be running in a Bundler context, whereas I initially ran `bridgetown` without prefixing it with `bundle exec`?**

**Comment by davidstosik** [comment]
It might be interesting to look at Rails to understand how the gem's `rails` command will hook into the current Rails app's `bin/rails`, if it happens to be running within a Rails app.

**Comment by jaredcwhite** [comment]
Shouldn't crash anymore if the rake gem isn't available with PR #656 merged.

**Comment by davidstosik** [comment]
Confirmed it's fixed in v1.2.0beta4. Thank you!
